changelogs/unreleased/gh-9556-fix-uri-ipv6.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/lua/uri
+
```

Comment:
commit message "uri: uri.format encapsulate IPv6 in []":

replace "uri: uri.format encapsulate IPv6 in []" with "uri: uri.format encapsulate IPv6 address in []"

Reply:
Replaced

Reply:
Thanks!